fxapom/pages/marionette/base.py

```diff
@@ -0,0 +1,16 @@
+
+from fxapom.fxapom import TIMEOUT
+from page import Page
+
+
+class Base(Page):
+    def __init__(self, driver, timeout=TIMEOUT):
+        super(Page, self).__init__(driver, timeout)
+        self.driver = driver
+        self._main_window_handle = self.driver.current_window_handle
+
+    def switch_to_main_window(self):
+        self.driver.switch_to_window(self._main_window_handle)
+
+    def close_window(self):
+        self.driver.close()
```

Comment:
This file doesn't appear to be any different from `fxapom/pages/base.py`, so it may not even be needed.